src/components/menu/menu.scss

```diff
@@ -256,7 +256,7 @@ $dark-blue: #294374;
       .MainMenu__submenu-container {
         background: $color-white;
         position: fixed;
-        top: 2.5rem;
+        top: 5rem;
```

Comment:
I guess this is to make the sub menu appear right below the main menu in case navigation items takes up 2 rows?
Unfortunately that would prevent user from accessing the top row's sub-menus because the mouse focus is lost when user hovers over the gap in between the main menu item and its submenu.

![image](https://user-images.githubusercontent.com/6004147/135572918-6146ca24-fa53-4a9a-a729-ba87b85cf4c6.png)

We could try to come up with a fix to accommodate cases where the navigation items flows to 2 lines. On the other hand, from a design angle, I suppose the main navigation shouldn't flow into 2 lines in the first place :sweat_smile:  We  actually discussed this with Hanni before, so that the content admin would try to make the menu item always fit in 1 line before the mobile snap point starts.

Another temporary alternative I can think of is to introduce another snap point, at which the main-nav changes to hamburger navigation.

Reply:
Actually, on my screen, it always goes to two lines, and Ajankohtaista is always on the second line 😅  Thanks for testing that! I have some ideas for the submenu. Will work with this during the weekend :) 

But yeah, we need to think some kind of solution for the two row-thing, but I would maybe keep it as it is for now, because it is good to have all of the items visible on desktop, to see an overview of the content of the page 🤔